pyanaconda/network.py

```diff
@@ -651,17 +651,15 @@ def _add_slave_connection(slave_type, slave, master, activate, values=None):
     mac = nm.nm_device_perm_hwaddress(slave)
     mac = [int(b, 16) for b in mac.split(":")]
     values.append(['802-3-ethernet', 'mac-address', mac, 'ay'])
+    # HACK preventing NM to autoactivate the connection
```

Comment:
If I understand it correctly, please correct me if not, the `ifconf` file will be changed later to have `ONBOOT=yes`. I think it would be nice to write it here.